File: aws/AWS_Lambda_in_Action/Chapter13/runLocal.js
var lambdaFunction = require('./greetingsOnDemand');
var functionHandler = 'handler';

var event = {}; // { name: 'Danilo'};
var context = {};

function callback(error, data) {
  console.log(error);
  console.log(data);
}

lambdaFunction[functionHandler](event, context, callback);

File: aws/AWS_Lambda_in_Action/Chapter13/runLocal.py
import importlib

mod = importlib.import_module('greetingsOnDemand')
functionHandler = 'lambda_handler'
lambdaFunction = getattr(mod, functionHandler)

event = { 'name' : 'Danilo' }
context = {}

try:
    data = lambdaFunction(event, context)
    print data
except Exception as error:
    print error
